pkg/model/crd.go

```diff
@@ -106,6 +107,18 @@ func (r *CRD) TypeRenames() map[string]string {
 	return r.sdkAPI.GetTypeRenames(r.cfg)
 }
 
+// Documentation returns the base documentation string for the API formatted as
+// a Go code comment block
+func (r *CRD) Documentation() string {
+	docString := fmt.Sprintf("// %sSpec defines the desired state of %s.", r.Names.Original, r.Names.Original)
+	shape, ok := r.sdkAPI.API.Shapes[r.Names.Original]
+	if ok {
+		// Separate with a double newline to force a newline in the CRD base
+		docString += "\n//\n" + shape.Documentation
+	}
+	return docString
```

Comment:
you may want to output a warning or something if there is no shape in the `API.Shapes` collection that matches the name of the resource...